docs/DeveloperGuide.md

```diff
@@ -49,7 +49,379 @@ Project team leaders can:
 | `*`       | expert user       | have the choice of deleting users from the app when a project completes        | quickly declutter my app                                                               |
 | `*`       | expert user       | set timers to add/delete groups after a project ends                           | ensure that I do not have too many groups cluttering the database                      |
 
+---
```

Comment:
I have made an edit to bring the extension lists in line with the ordered lists, and will update this file with my corrections. Let me know if there are any errors then.